pkgs/by-name/ra/rain-bittorrent/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule {
+  pname = "rain";
+  version = "1.12.17";
```

```suggestion
  version = "1.13.0";
```
New version has been released by now.